klimenko/Lesson14/src/main/java/com/klimenko/Lesson14/entity/Apartment.java

```diff
@@ -0,0 +1,172 @@
+package com.klimenko.Lesson14.entity;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "apartments")
+public class Apartment {
+
+    @NotNull
+    private int floor;
+
+    @NotNull
+    @Min(value = 1, message = "Количество комнат не может быть меньше 1")
+    private int countOfRoom;
+
+    @NotNull
+    @Min(value = 1, message = "Размер не может быть меньше 1")
+    private int size;
+
+    private String address;
+    private String typeEstate;
+    private String date;
```

Comment:
It's not a good approach, to save date as a string. Better save Date, and convert it, if you need String